### PR TITLE
Added CH20 to the open source events

### PR DESCRIPTION
**Pull request by AdityaJ7**
# Description

Added CH20 to the open-source events folder.

Fixes  issue #265 

## Type of change

- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Open_Source_Events/CH20/2020.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Stats of STUDENT CODE-IN (SCI) 2020
```
Comment (HarshCasper) [inline Open_Source_Events/CH20/2020.md:1]: Remove this Document

**Review by HarshCasper — APPROVED** [review]
Please make the necessary change

File: Open_Source_Events/CH20/2020.md (hunk 1 above)
Comment (AdityaJ7) [reply Open_Source_Events/CH20/2020.md]: done!

**Comment by AdityaJ7** [comment]
> Please make the necessary change

Made the changes.

**Pull request merged**